docs/checkit.js

```diff
@@ -623,29 +632,32 @@ CheckIt.prototype.createCalendar = function() {
     if (this.validateForm(start, end)) {
         
         //clear the previously displayed calendar
-        this.clearPage(this.$calendarDiv);
+        this.clearPage();
 
         //make a calendar State
         var state = emptyCalendarState({startDate: start, endDate: end, calendarTitle: title});
         
         //make calendar object
 
         var calendar = new Calendar(state);
+        // TODO figure out if I should be returning initializeCalendar and buildCalendar promises
```

Comment:
I'd say so. Without doing it you can't write any code that depends on this being done. There's no real reason not to.